reference/features/telemetry.md

```diff
@@ -0,0 +1,76 @@
+# Telemetry
+
+MeiliSearch collects anonymized data from users in order to improve our product. This can be [deactivated at any time](#how-to-disable-data-collection).
+
+Note that while MeiliSearch is running in [`development` mode](/reference/features/configuration.md#environment), no data is collected.
+
+## Why collect telemetry data?
+
+We collect anonymized data for only two reasons: so that we can improve our product, and so that we can continue working on this project full-time.
+
+In order to better address our users' needs and chart a clear path forward for MeiliSearch, we need reliable quantitative information. The information we collect helps us fix bugs, evaluate the success of certain features, and identify our strengths and weaknesses as a product.
+
+Usage metrics also help us justify our existence to investors so that we can keep the MeiliSearch project alive. They prove what we all believe: that MeiliSearch is a great product that can be of use to a large number of people around the world.
+
+## What kind of data do we collect?
```

Comment:
Would it be better to say **what** we collect first and **why** we collect it later? I'm thinking a user might care more about what is being collected than why. 

Personally I will never believe a company when they say they only collect data for benign purposes and assume they will eventually sell it to someone; hence I would be more interested in seeing what I will be giving them.